mesonbuild/compilers/mixins/visualstudio.py

```diff
@@ -26,6 +26,7 @@
 
 if T.TYPE_CHECKING:
     from ...environment import Environment
+    from ..dependencies import Dependency
```

Comment:
this should be:
```python
from ...dependencies import Dependency
```

That will make the type checker happy I think.

Reply:
Right, thanks.